### DISCUSSION TITLE
How to do updates where patches are in upstream pull request

### DISCUSSION THREAD

**Discussion by areese** [Homebrew/discussions#1565] [Tap maintenance and Homebrew development] [answered]
I've noticed that homebrew prefers Formula that the upstream is patched.
I've seen a method where a specific patch can be pull by commit, does this work for open PR's?

The reason I ask is I have opened a PR for fluent-bit to support m1: https://github.com/fluent/fluent-bit/pull/3564

I would like to PR the fluent-bit formula, and would like to know if that PR needs to be closed before proceeding.

**Comment by carlocab** [comment, marked as answer]
You can use the patch from the commit in your PR (i.e. https://github.com/fluent/fluent-bit/commit/1f682a8d07ba649ae0f92d326a08967d59a9f465.patch?full_index=1)

**Reply by areese** [reply]
Thank you I’ll do that
